Mohammad-Husam/coding_challenge
# building text input fields

**Background**

-  Building employee text input fields.

**Modifications**

- using flutter_screenutil 5.0.0+2  flutter plugin for adapting screen and font size. https://pub.dev/packages/flutter_screenutil 
- building employee fields without any operations.
- adding single child scroll view to screen to make it scrollable.

**Step to test**

1. open the app.
2. observe the result on the first page of the app.

- you can rotate the screen to check if the screen is scrollable or not.
